ginkgo_linter.go

```diff
@@ -312,7 +316,8 @@ func checkNilError(pass *analysis.Pass, assertionExp *ast.CallExpr, handler gome
 	return false
 }
 
-func checkEqualNil(pass *analysis.Pass, assertionExp *ast.CallExpr, handler gomegahandler.Handler, actualArg ast.Expr, oldExpr string) bool {
+// handle Equal(nil), Equal(true) and Equal(false)
+func checkEqualSpecial(pass *analysis.Pass, exprSuppress types.Suppress, assertionExp *ast.CallExpr, handler gomegahandler.Handler, actualArg ast.Expr, oldExpr string) bool {
```

Comment:
`checkEqualSpecial` -> `checkEqualAssertion`/Matcher?

Reply:
I agree that we can use a better name here, but this is not the only place we check the `Equal` assertion. This one is for assertion of golang reserved words `nil`, `true` and `false`.

Reply:
OK, I changed it to `simplifyEqaul`.